packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Models/Types/MrwSerializationTypeProvider.cs

```diff
@@ -85,13 +86,13 @@ protected override CSharpType[] BuildImplements()
         /// </summary>
         internal CSharpMethod BuildJsonModelWriteMethod()
         {
-            Parameter utf8JsonWriterParameter = new("writer", null, typeof(Utf8JsonWriter), null, ValidationType.None, null);
+            Parameter utf8JsonWriterParameter = new("writer", null, typeof(Utf8JsonWriter), null, ParameterValidationType.None, null);
             // void IJsonModel<T>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
             return new CSharpMethod
             (
               new MethodSignature(nameof(IJsonModel<object>.Write), null, null, MethodSignatureModifiers.None, null, null, new[] { utf8JsonWriterParameter, SerializationOptionsParameter }, ExplicitInterface: _iJsonModelTInterface),
               // TO-DO: Add body for json properties' serialization https://github.com/microsoft/typespec/issues/3330
-              Snippets.EmptyStatement
+              Snippet.EmptyStatement
```

Comment:
I like `Snippets` more becuase it is a big static class which provides quite a few snippets